src/Map.cpp

```diff
@@ -65,22 +65,22 @@ void cMap::Tick()
 
 
 
-void cMap::UpdateRadius(int a_PixelX, int a_PixelZ, unsigned int a_Radius)
+void cMap::UpdateRadius(unsigned int a_PixelX, unsigned int a_PixelZ, unsigned int a_Radius)
 {
-	int PixelRadius = a_Radius / GetPixelWidth();
+	unsigned int PixelRadius = a_Radius / GetPixelWidth();
 
-	unsigned int StartX = Clamp(a_PixelX - PixelRadius, 0, (int)m_Width);
-	unsigned int StartZ = Clamp(a_PixelZ - PixelRadius, 0, (int)m_Height);
+	unsigned int StartX = Clamp(a_PixelX - PixelRadius, static_cast<unsigned int>(0), m_Width);
+	unsigned int StartZ = Clamp(a_PixelZ - PixelRadius, static_cast<unsigned int>(0), m_Height);
```

Comment:
They used to be `int` in the original code, so the expression `a_PixelX - PixelRadius` could have been negative, the `Clamp` would then clamp it to zero. Now, the expression is positive (huge number) and `Clamp` clamps it to `m_Width`